roles/gNB/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+srsran:
+  docker:
+    container:
+      gnb_image: srsran-gnb:latest
```

Comment:
Should you update this? Such image does not exist, correct?

Reply:
done